recipes/vmtk/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "vmtk" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vmtk/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ec52357bf6dfd975eaae592b6b70db67c7767a01278386373588a29cf52997bb
+
+build:
+  number: 0
+  skip:
+    - True  # [win and py27]
+    - True  # [py37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkgconfig              # [unix]
+    - cmake
+    - git
+    - ninja
+    - setuptools             # [win]
+
+  host:
+    - python
+    - future
+    - vtk <= 8.1.0
+    - libitk
+    - numpy
+    - h5py
+    - setuptools              # [win]
+    - mesa                    # [linux]
+    - joblib
```

Comment:
Are all of these needed to build the package, or only to run it? Only items needed to build the package should be here.